models/db_pull.js

```diff
@@ -12,6 +12,7 @@ function DBPull(pull) {
       state: pullData.state,
       title: pullData.title,
       body: pullData.body,
+      draft: pullData.draft,
```

Comment:
I think you'll also need to add this column to the schema. Looks like we normally add a migration in the `migrations` directory, and it seems we also update the `schema.sql` file (not sure if that's manual, or automatic somehow):
https://github.com/iFixit/pulldasher/tree/draft-dev-block/migrations

Here's a commit that seems to be doing the thing:
https://github.com/iFixit/pulldasher/commit/ae18be0a17f7c7d5896bd6aca1c7a76b1f0205b0

Reply:
It's manual.

Reply:
Updated schema.sql file via https://github.com/iFixit/pulldasher/pull/329/commits/03023cbd41d23d33d7814906697fd9957a137496

Reply:
Column added via https://github.com/iFixit/pulldasher/pull/329/commits/1fc5deb414751b1a79f666cb4cd41074bae04326